include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -59,24 +60,6 @@ class device_policy
         return q;
     }
 
-    // For internal use only
-    static constexpr ::std::true_type
```

Comment:
@rarutyun Will we have here ABI breaking change?
We remove methods from public class...

Reply:
Agree